chore: lint check fails on master

A recent change of `flake8` or its config causes the code style check to fail on the `master` branch - it complains about a variable name in a line in `job.py` that has last been changed almost 10 months ago:
```
google/cloud/bigquery/job.py:3119:39: E741 ambiguous variable name 'l'
```

--- a/google/cloud/bigquery/job.py
+++ b/google/cloud/bigquery/job.py
@@ -3116,7 +3116,7 @@ class QueryJob(_AsyncJob):
         template = "\n\n(job ID: {job_id})\n\n{header}\n\n{ruler}\n{body}\n{ruler}"
 
         lines = query.splitlines()
-        max_line_len = max(len(l) for l in lines)
+        max_line_len = max(len(line) for line in lines)
 
         header = "-----Query Job SQL Follows-----"
         header = "{:^{total_width}}".format(header, total_width=max_line_len + 5)

